rules/windows/command_and_control_sunburst_c2_activity_detected.toml

```diff
@@ -0,0 +1,80 @@
+[metadata]
+creation_date = "2020/12/14"
+maturity = "production"
+updated_date = "2020/12/14"
+
+[rule]
+author = ["Elastic"]
+description = """
+The malware known as SUNBURST targets the SolarWind's Orion business software for command and control. This rule detects
+post-exploitation command and control activity of the SUNBURST backdoor.
+"""
+from = "now-9m"
+index = ["logs-endpoint.events.*"]
+language = "kuery"
+license = "Elastic License"
+name = "SUNBURST Command and Control Activity"
+note = "The SUNBURST malware attempts to hide within the Orion Improvement Program (OIP) network traffic. As this rule detects post-exploitation network traffic, investigations into this should be prioritized."
+references = [
+    "https://www.fireeye.com/blog/threat-research/2020/12/evasive-attacker-leverages-solarwinds-supply-chain-compromises-with-sunburst-backdoor.html",
+]
+risk_score = 73
+rule_id = "22599847-5d13-48cb-8872-5796fee8692b"
+severity = "high"
+tags = ["Elastic", "Host", "Windows", "Threat Detection", "Command and Control"]
+type = "query"
+
+query = '''
+event.category:network and event.type:protocol and network.protocol:http and
+	process.name:(
+		ConfigurationWizard.exe or
+		NetFlowService.exe or
+		NetflowDatabaseMaintenance.exe or
+		SolarWinds.Administration.exe or
+		SolarWinds.BusinessLayerHost.exe or
+		SolarWinds.BusinessLayerHostx64.exe or
+		SolarWinds.Collector.Service.exe or
+		SolarwindsDiagnostics.exe) and
+  http.request.body.content:((
```

Comment:
Making a future note that we may consider an OR expression between the http.request.body.content and a generic non-RFC1918 network destination, although it is likely the vendor will make drastic modifications to their network behavior as a result of this incident. Adding an OR expression may enable compatibility for the users deploying Sysmon and WinLogBeat. Sysmon, lacking a feature to inspect web content, may therefore be considered a poor substitute.